### PR TITLE
Chapter 2 proofreading edits

### PR DESCRIPTION
**Pull request by psywombats**
Nothing too much here, short chapter. Just technical stuff really.

### CODE REVIEW

File: chapter02/chapter02.rpy [hunk 1]
```diff
@@ -52,88 +52,86 @@ label chapter_02:
...
     The woman walked forward to me and a placed the tray of food on top of my lap, causing me to tense up at the sight of it.
-    Nurse "Well, you were asleep for a while so you missed dinner. [pause] Go ahead and eat it."
+    m "Well, you were asleep for a while so you missed dinner. [Pause] Go ahead and eat it."
     The tray consisted of a large bowl of steaming rice, a plate decked with some sort of glistening meat and vegetables, and a cup with some sort of warm white liquid in it.
     I shivered at the sight of the food, all my memories flooding into me at once.
     I picked up the tray with my trembling right hand and threw it past the woman, causing a loud clattering sound to echo inside the room.
     The woman gasped at my actions, turning her attention to the mess near the door, where the tray had landed.
     After a minute or two of silence from the woman, she turned around and glared at me with frustrated and angry eyes. 
     I glared back, taking the action as some sort of threatening position.
-    Nurse "What in the name of Ion makes you think that you have the authority to just throw a perfectly well made meal to the wall like that?!"
```
Comment (psywombats) [inline chapter02/chapter02.rpy:131]: I've changed these to Meadow so that in the script she'll speak with the same color and portrait.

Comment (Phantom-Chan) [inline chapter02/chapter02.rpy:131]: I'm okay with changing the "Nurse" to "Meadow" to keep the color and portrait, but at the same time I'm not so sure. There is an option later on that will effect the ending the player will get where the player (as Kenta) will decide whether or not they trust Meadow. By saying yes, Meadow reintroduces herself and from the point on in Kenta's perspective she will be known as Meadow. If no, she will remain as "Nurse" to him. If it's possible to change Nurse to Meadow while having the name be _shown_ as Nurse, then it's okay. I've only worked with ren'py for a little bit and I know the basics. I'm not sure if this is possible, but you've had more experience than me so maybe there's a way.

Comment (psywombats) [inline chapter02/chapter02.rpy:131]: Yep! That should definitely be possible, thanks for explaining your reasoning. I guess this is what I get for not reading the entire script through first. I'll correct this when I get home.

Comment (psywombats) [inline chapter02/chapter02.rpy:131]: @Phantom-Chan Changed it back. Should be fixed now.

**Pull request merged**